### ISSUE TITLE
git merge did not merge newfeature branch

### ISSUE THREAD

**Issue by lisamsmith100** [ga-wdi-boston/game-project#562]
This is a continuation of issue from yesterday, working with Ben.
My commands:

git commit (newfeature) branch
git push origin newfeature
git checkout master
git push origin master
git merge newfeature
grunt deploy

pasting my terminal code in a moment.

Yesterday, we talked about maybe needing to delete gh-pages and grunt deploy if the grunt deploy did not work.

**Comment by lisamsmith100** [comment]
Pasted part of what is in the terminal:
```
$ grunt deploy
Running "shell:git-is-clean" (shell) task

Running "shell:git-checkout-master" (shell) task
Already on 'master'

Running "webpack:build" (webpack) task
Version: webpack 1.14.0     
                                 Asset     Size  Chunks             Chunk Names
  f4769f9bdb7466be65088239c12046d1.eot  20.1 kB          [emitted]  
448c34a56d699c29117adc64c43affeb.woff2    18 kB          [emitted]  
 fa2772327f55d8198301fdb8bcfc8158.woff  23.4 kB          [emitted]  
  e18bbf611f2a2e43afc071aa2f4e1512.ttf  45.4 kB          [emitted]  
  89889688147bd7575d6327160d64e760.svg   109 kB          [emitted]  
                        application.js   184 kB       0  [emitted]  application
                              specs.js  1.17 kB       1  [emitted]  specs
                             vendor.js   352 kB       2  [emitted]  vendor
   [0] ./index.js 248 bytes {0} [built]
   [0] ./spec/_all.js 198 bytes {1} [built]
   [0] multi vendor 40 bytes {2} [built]
   [1] ./assets/scripts/index.js 301 bytes {0} [built]
       cjs require ./assets/scripts/index.js [0] ./index.js 9:0-36
   [2] ./~/jquery/dist/jquery.js 268 kB {2} [built]
       single entry jquery [0] multi vendor
       cjs require $ [1] ./assets/scripts/index.js 1:0-17
       cjs require $ [7] ./assets/scripts/auth/events.js 1:0-17
       cjs require $ [9] ./assets/scripts/auth/api.js 1:0-17
       cjs require $ [11] ./assets/scripts/auth/ui.js 1:0-17
       cjs require $ [12] ./assets/scripts/game/events.js 1:0-17
       cjs require $ [13] ./assets/scripts/game/api.js 1:0-17
       cjs require $ [14] ./assets/scripts/game/ui.js 1:0-17
       cjs require jQuery [27] ./~/bootstrap-sass/assets/javascripts/bootstrap.js 1:0-17
   [3] ./lib/set-api-origin.js 1.11 kB {0} [built]
       cjs require ../../lib/set-api-origin [1] ./assets/scripts/index.js 3:19-54
   [4] ./lib/parse-nested-query.js 403 bytes {0} [built]
       cjs require ./parse-nested-query [3] ./lib/set-api-origin.js 3:23-54
   [5] ./lib/add-nested-value.js 745 bytes {0} [built]
       cjs require ./add-nested-value [4] ./lib/parse-nested-query.js 3:21-50
       cjs require ./add-nested-value [8] ./lib/get-form-fields.js 3:21-50
   [6] ./assets/scripts/config.js 139 bytes {0} [built]
       cjs require ./config [1] ./assets/scripts/index.js 4:13-32
       cjs require ../config [9] ./assets/scripts/auth/api.js 3:13-33
       cjs require ../config [13] ./assets/scripts/game/api.js 3:13-33
   [7] ./assets/scripts/auth/events.js 1.23 kB {0} [built]
       cjs require ./auth/events [1] ./assets/scripts/index.js 5:17-41
   [8] ./lib/get-form-fields.js 960 bytes {0} [built]
       cjs require ../../../lib/get-form-fields [7] ./assets/scripts/auth/events.js 3:20-59
       cjs require ../../../lib/get-form-fields [12] ./assets/scripts/game/events.js 3:20-59
   [9] ./assets/scripts/auth/api.js 1.02 kB {0} [built]
       cjs require ./api [7] ./assets/scripts/auth/events.js 5:10-26
  [10] ./assets/scripts/store.js 55 bytes {0} [built]
       cjs require ../store [9] ./assets/scripts/auth/api.js 4:12-31
       cjs require ../store [11] ./assets/scripts/auth/ui.js 3:12-31
       cjs require ../store [13] ./assets/scripts/game/api.js 4:12-31
       cjs require ../store [14] ./assets/scripts/game/ui.js 3:12-31
  [11] ./assets/scripts/auth/ui.js 1.88 kB {0} [built]
       cjs require ./ui [7] ./assets/scripts/auth/events.js 6:9-24
  [12] ./assets/scripts/game/events.js 1.14 kB {0} [built]
       cjs require ./game/events [1] ./assets/scripts/index.js 6:17-41
       cjs require ./events [13] ./assets/scripts/game/api.js 5:17-36
  [13] ./assets/scripts/game/api.js 1.2 kB {0} [built]
       cjs require ./api [12] ./assets/scripts/game/events.js 4:10-26
  [14] ./assets/scripts/game/ui.js 1.67 kB {0} [built]
       cjs require ./ui [12] ./assets/scripts/game/events.js 5:9-24
  [15] ./assets/styles/index.scss 1.03 kB {0} [built]
       cjs require ./assets/styles/index.scss [0] ./index.js 12:0-37
  [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 160 kB {0} [built]
       cjs require !!../../node_modules/css-loader/index.js!../../node_modules/sass-loader/index.js!./index.scss [15] ./assets/styles/index.scss 4:14-118
  [17] ./~/css-loader/lib/css-base.js 1.51 kB {0} [built]
       cjs require ../../node_modules/css-loader/lib/css-base.js [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 1:27-83
  [18] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.eot 82 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.eot [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4172-4253
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.eot [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4276-4357
  [19] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.woff2 84 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.woff2 [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4409-4492
  [20] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.woff 83 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.woff [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4525-4607
  [21] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.ttf 82 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.ttf [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4639-4720
  [22] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.svg 82 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.svg [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4756-4837
  [23] ./~/style-loader/addStyles.js 6.91 kB {0} [built]
       cjs require !../../node_modules/style-loader/addStyles.js [15] ./assets/styles/index.scss 7:13-69
  [24] ./spec spec\.js$ 182 bytes {1} [built]
       require.context ./ [0] ./spec/_all.js 6:10-50
  [25] ./spec/example.spec.js 173 bytes {1} [optional] [built]
       context element ./example.spec.js [24] ./spec spec\.js$
  [26] ./assets/scripts/example.js 37 bytes {1} [built]
       cjs require ../assets/scripts/example [25] ./spec/example.spec.js 3:14-50
  [27] ./~/bootstrap-sass/assets/javascripts/bootstrap.js 69.7 kB {2} [built]
       single entry bootstrap-sass [0] multi vendor

Running "shell:git-push-master" (shell) task
Everything up-to-date

Running "shell:deploy-prepare" (shell) task
Deleted branch gh-pages (was aaacda22).
Switched to a new branch 'gh-pages'
rm 'CONTRIBUTING.md'
rm 'Gruntfile.js'
rm 'LICENSE'
rm 'README.md'
rm 'STYLE.md'
rm 'assets/scripts/.eslintrc.json'
rm 'assets/scripts/apps.js'
rm 'assets/scripts/auth/api.js'
rm 'assets/scripts/auth/events.js'
rm 'assets/scripts/auth/ui.js'
rm 'assets/scripts/config.js'
rm 'assets/scripts/example.js'
rm 'assets/scripts/game/api.js'
rm 'assets/scripts/game/events.js'
rm 'assets/scripts/game/logic.js'
rm 'assets/scripts/game/ui.js'
rm 'assets/scripts/gameEvents/game.js'
rm 'assets/scripts/index.js'
rm 'assets/scripts/store.js'
rm 'assets/scripts/templates/helpers/.keep'
rm 'assets/styles/index.scss'
rm 'assets/styles/tacoImage.png'
rm 'assets/styles/tic-tac-to.html'
rm 'assets/sty

Running "webpack:build" (webpack) task
Version: webpack 1.14.0     
                                 Asset     Size  Chunks             Chunk Names
  f4769f9bdb7466be65088239c12046d1.eot  20.1 kB          [emitted]  
448c34a56d699c29117adc64c43affeb.woff2    18 kB          [emitted]  
 fa2772327f55d8198301fdb8bcfc8158.woff  23.4 kB          [emitted]  
  e18bbf611f2a2e43afc071aa2f4e1512.ttf  45.4 kB          [emitted]  
  89889688147bd7575d6327160d64e760.svg   109 kB          [emitted]  
                        application.js   184 kB       0  [emitted]  application
                              specs.js  1.17 kB       1  [emitted]  specs
                             vendor.js   352 kB       2  [emitted]  vendor
   [0] ./index.js 248 bytes {0} [built]
   [0] ./spec/_all.js 198 bytes {1} [built]
   [0] multi vendor 40 bytes {2} [built]
   [1] ./assets/scripts/index.js 301 bytes {0} [built]
       cjs require ./assets/scripts/index.js [0] ./index.js 9:0-36
   [2] ./~/jquery/dist/jquery.js 268 kB {2} [built]
       single entry jquery [0] multi vendor
       cjs require $ [1] ./assets/scripts/index.js 1:0-17
       cjs require $ [7] ./assets/scripts/auth/events.js 1:0-17
       cjs require $ [9] ./assets/scripts/auth/api.js 1:0-17
       cjs require $ [11] ./assets/scripts/auth/ui.js 1:0-17
       cjs require $ [12] ./assets/scripts/game/events.js 1:0-17
       cjs require $ [13] ./assets/scripts/game/api.js 1:0-17
       cjs require $ [14] ./assets/scripts/game/ui.js 1:0-17
       cjs require jQuery [27] ./~/bootstrap-sass/assets/javascripts/bootstrap.js 1:0-17
   [3] ./lib/set-api-origin.js 1.11 kB {0} [built]
       cjs require ../../lib/set-api-origin [1] ./assets/scripts/index.js 3:19-54
   [4] ./lib/parse-nested-query.js 403 bytes {0} [built]
       cjs require ./parse-nested-query [3] ./lib/set-api-origin.js 3:23-54
   [5] ./lib/add-nested-value.js 745 bytes {0} [built]
       cjs require ./add-nested-value [4] ./lib/parse-nested-query.js 3:21-50
       cjs require ./add-nested-value [8] ./lib/get-form-fields.js 3:21-50
   [6] ./assets/scripts/config.js 139 bytes {0} [built]
       cjs require ./config [1] ./assets/scripts/index.js 4:13-32
       cjs require ../config [9] ./assets/scripts/auth/api.js 3:13-33
       cjs require ../config [13] ./assets/scripts/game/api.js 3:13-33
   [7] ./assets/scripts/auth/events.js 1.23 kB {0} [built]
       cjs require ./auth/events [1] ./assets/scripts/index.js 5:17-41
   [8] ./lib/get-form-fields.js 960 bytes {0} [built]
       cjs require ../../../lib/get-form-fields [7] ./assets/scripts/auth/events.js 3:20-59
       cjs require ../../../lib/get-form-fields [12] ./assets/scripts/game/events.js 3:20-59
   [9] ./assets/scripts/auth/api.js 1.02 kB {0} [built]
       cjs require ./api [7] ./assets/scripts/auth/events.js 5:10-26
  [10] ./assets/scripts/store.js 55 bytes {0} [built]
       cjs require ../store [9] ./assets/scripts/auth/api.js 4:12-31
       cjs require ../store [11] ./assets/scripts/auth/ui.js 3:12-31
       cjs require ../store [13] ./assets/scripts/game/api.js 4:12-31
       cjs require ../store [14] ./assets/scripts/game/ui.js 3:12-31
  [11] ./assets/scripts/auth/ui.js 1.88 kB {0} [built]
       cjs require ./ui [7] ./assets/scripts/auth/events.js 6:9-24
  [12] ./assets/scripts/game/events.js 1.14 kB {0} [built]
       cjs require ./game/events [1] ./assets/scripts/index.js 6:17-41
       cjs require ./events [13] ./assets/scripts/game/api.js 5:17-36
  [13] ./assets/scripts/game/api.js 1.2 kB {0} [built]
       cjs require ./api [12] ./assets/scripts/game/events.js 4:10-26
  [14] ./assets/scripts/game/ui.js 1.67 kB {0} [built]
       cjs require ./ui [12] ./assets/scripts/game/events.js 5:9-24
  [15] ./assets/styles/index.scss 1.03 kB {0} [built]
       cjs require ./assets/styles/index.scss [0] ./index.js 12:0-37
  [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 160 kB {0} [built]
       cjs require !!../../node_modules/css-loader/index.js!../../node_modules/sass-loader/index.js!./index.scss [15] ./assets/styles/index.scss 4:14-118
  [17] ./~/css-loader/lib/css-base.js 1.51 kB {0} [built]
       cjs require ../../node_modules/css-loader/lib/css-base.js [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 1:27-83
  [18] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.eot 82 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.eot [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4172-4253
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.eot [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4276-4357
  [19] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.woff2 84 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.woff2 [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4409-4492
  [20] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.woff 83 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.woff [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4525-4607
  [21] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.ttf 82 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.ttf [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4639-4720
  [22] ./~/bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.svg 82 bytes {0} [built]
       cjs require bootstrap-sass/assets/fonts/bootstrap/glyphicons-halflings-regular.svg [16] ./~/css-loader!./~/sass-loader!./assets/styles/index.scss 6:4756-4837
  [23] ./~/style-loader/addStyles.js 6.91 kB {0} [built]
       cjs require !../../node_modules/style-loader/addStyles.js [15] ./assets/styles/index.scss 7:13-69
  [24] ./spec spec\.js$ 182 bytes {1} [built]
       require.context ./ [0] ./spec/_all.js 6:10-50
  [25] ./spec/example.spec.js 173 bytes {1} [optional] [built]
       context element ./example.spec.js [24] ./spec spec\.js$
  [26] ./assets/scripts/example.js 37 bytes {1} [built]
       cjs require ../assets/scripts/example [25] ./spec/example.spec.js 3:14-50
  [27] ./~/bootstrap-sass/assets/javascripts/bootstrap.js 69.7 kB {2} [built]
       single entry bootstrap-sass [0] multi vendor

Running "shell:deploy-publish" (shell) task
.nojekyll { dev: 16777220,
  mode: 33188,
  nlink: 1,
  uid: 501,
  gid: 20,
  rdev: 0,
  blksize: 4096,
  ino: 5204690,
  size: 0,
  blocks: 0,
  atime: 2017-04-07T20:49:36.000Z,
  mtime: 2017-04-07T20:49:36.000Z,
  ctime: 2017-04-07T20:49:36.000Z,
  birthtime: 2017-04-07T20:49:36.000Z }
[gh-pages (root-commit) ce8e5ff7] deploy task
 11 files changed, 14188 insertions(+)
 create mode 100644 .nojekyll
 create mode 100644 favicon.ico
 create mode 100644 index.html
 create mode 100644 public/448c34a56d699c29117adc64c43affeb.woff2
 create mode 100644 public/89889688147bd7575d6327160d64e760.svg
 create mode 100644 public/application.js
 create mode 100644 public/e18bbf611f2a2e43afc071aa2f4e1512.ttf
 create mode 100644 public/f4769f9bdb7466be65088239c12046d1.eot
 create mode 100644 public/fa2772327f55d8198301fdb8bcfc8158.woff
 create mode 100644 public/specs.js
 create mode 100644 public/vendor.js
To github.com:lisamsmith100/tic-tac-toe-client.git
 + f539988d...ce8e5ff7 gh-pages -> gh-pages (forced update)
Removing CONTRIBUTING.md
Removing Gruntfile.js
Removing
Running "shell:deploy-publish" (shell) task
.nojekyll { dev: 16777220,
  mode: 33188,
  nlink: 1,
  uid: 501,
  gid: 20,
  rdev: 0,
  blksize: 4096,
  ino: 5204690,
  size: 0,
  blocks: 0,
  atime: 2017-04-07T20:49:36.000Z,
  mtime: 2017-04-07T20:49:36.000Z,
  ctime: 2017-04-07T20:49:36.000Z,
  birthtime: 2017-04-07T20:49:36.000Z }
[gh-pages (root-commit) ce8e5ff7] deploy task
 11 files changed, 14188 insertions(+)
 create mode 100644 .nojekyll
 create mode 100644 favicon.ico
 create mode 100644 index.html
 create mode 100644 public/448c34a56d699c29117adc64c43affeb.woff2
 create mode 100644 public/89889688147bd7575d6327160d64e760.svg
 create mode 100644 public/application.js
 create mode 100644 public/e18bbf611f2a2e43afc071aa2f4e1512.ttf
 create mode 100644 public/f4769f9bdb7466be65088239c12046d1.eot
 create mode 100644 public/fa2772327f55d8198301fdb8bcfc8158.woff
 create mode 100644 public/specs.js
 create mode 100644 public/vendor.js
To github.com:lisamsmith100/tic-tac-toe-client.git
 + f539988d...ce8e5ff7 gh-pages -> gh-pages (forced update)
Removing CONTRIBUTING.md
Removing Gruntfile.js
Removing LICENSE
Removing README.md
Removing STYLE.md
Removing assets/
Removing browser-template-master.zip
Removing forms.md
Removing grunt/
Removing index.js
Removing lib/
Removing package.json
Removing scripts/
Removing spec/
error: The following untracked working tree files would be overwritten by checkout:
	node_modules/.bin/acorn
	node_modules/.bin/babylon
	node_modules/.bin/browserslist



Aborted due to warnings.


Execution Time (2017-04-07 16:49:30 UTC-4)
loading tasks              928ms  ▇▇▇▇▇▇▇▇ 11%
shell:git-is-clean         121ms  ▇▇ 1%
shell:git-checkout-master  172ms  ▇▇ 2%
webpack:build               2.6s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 31%
shell:git-push-master      548ms  ▇▇▇▇▇ 7%
shell:deploy-prepare       377ms  ▇▇▇▇ 5%
webpack:build               1.1s  ▇▇▇▇▇▇▇▇▇▇ 13%
shell:deploy-publish        2.5s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 30%
Total 8.3s

```

**Comment by lisamsmith100** [comment]
I'm still hung on this.  Can we take another look?

**Comment by jordanallain** [comment]
What's the issue deploy not working or branch not merging?

**Comment by payne-chris-r** [comment]
Did this get resolved? I know you were working with @BenGitsCode at the end of the day.

**Comment by BenGitsCode** [comment]
Antony walked @lisamsmith100 through fixing this. Can you please update the issue with resolution and close it?

**Comment by lisamsmith100** [comment]
Antony  and Ben spent time with  me on this issue.  Issue resolved, I think.  Thank you.  I owe you both a cocktail a yvonnes or somthings.  Thank you.

**Comment by BenGitsCode** [comment]
That's not a resolution. A good closing comment should include any new understanding of the problem and elucidations as to how it was resolved. 

You can see some good examples of closing comments in #515 and #501

**Comment by lisamsmith100** [comment]
The issue was that there were about 8 files that were missing from my project files and affected the deployment.

We cloned the master file from library-api-guide into a new branch, identified which files were missing (git diff).  Then we moved those files into my project file and rechecked if there were any differences.  When we did not see any differences, we added, committed, and merged to master branch then deployed.
